go.mod

```diff
@@ -21,13 +19,13 @@ require (
 	golang.org/x/net v0.30.0
 	golang.org/x/text v0.19.0
 	helm.sh/helm/v3 v3.16.2
-	k8s.io/api v0.31.2
+	k8s.io/api v0.32.2
 	k8s.io/apiextensions-apiserver v0.31.2
-	k8s.io/apimachinery v0.31.2
-	k8s.io/client-go v0.31.2
+	k8s.io/apimachinery v0.32.2
+	k8s.io/client-go v0.32.2
 	k8s.io/component-base v0.31.2
```

Comment:
bump?

Reply:
yea fixing it I need a moment, this is automatically being set to v0.31.2 I probably have to use replace directive.

Reply:
> this is automatically being set to v0.31.2 I probably have to use replace directive.

That doesn't seem right 🤔

Reply:
Didn't end up needing the replace, sorry I was in the middle of brainstorming session that time 😅